Aztec/Classes/Converters/HTMLNodeToNSAttributedString.swift

```diff
@@ -99,7 +99,7 @@ class HMTLNodeToNSAttributedString: SafeConverter {
     ///
     fileprivate func convertCommentNode(_ node: CommentNode, inheritingAttributes inheritedAttributes: [String:Any]) -> NSAttributedString {
         let moreLabel = "more"
-        if node.comment.hasPrefix(moreLabel) {
+        if node.comment.lowercased().hasPrefix(moreLabel) {
```

Comment:
At least in Calypso the more tag is only recognised if all the character are lowercase, if you write More or MORE it isn't pick up as a more separator.